packages/admin-e2e-tests/src/specs/homescreen/activity-panel.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import { createSimpleProduct, withRestApi } from '@woocommerce/e2e-utils';
+
+/**
+ * Internal dependencies
+ */
+import { Login } from '../../pages/Login';
+import { OnboardingWizard } from '../../pages/OnboardingWizard';
+import { WcHomescreen } from '../../pages/WcHomescreen';
+import { createOrder, removeAllOrders, updateOption } from '../../fixtures';
+import { OrdersActivityPanel } from '../../elements/OrdersActivityPanel';
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { afterAll, beforeAll, describe, it } = require( '@jest/globals' );
+/* eslint-enable @typescript-eslint/no-var-requires */
+
+const simpleProductName = 'Simple order';
+const testAdminHomescreenActivityPanel = () => {
+	describe( 'Homescreen activity panel', () => {
+		const profileWizard = new OnboardingWizard( page );
+		const homeScreen = new WcHomescreen( page );
+		const ordersPanel = new OrdersActivityPanel( page );
+		const login = new Login( page );
+
+		beforeAll( async () => {
+			await login.login();
+
+			// This makes this test more isolated, by always navigating to the
+			// profile wizard and skipping, this behaves the same as if the
+			// profile wizard had not been run yet and the user is redirected
+			// to it when trying to go to wc-admin.
+			await withRestApi.deleteAllProducts();
+			await removeAllOrders();
+			await updateOption( 'woocommerce_task_list_hidden', 'no' );
+			await profileWizard.navigate();
+			await profileWizard.skipStoreSetup();
+
+			await homeScreen.isDisplayed();
+			await homeScreen.possiblyDismissWelcomeModal();
+		} );
+
+		afterAll( async () => {
+			await withRestApi.deleteAllProducts();
+			await removeAllOrders();
+			await updateOption( 'woocommerce_task_list_hidden', 'no' );
+			await login.logout();
+		} );
+
+		it( 'should not show activity panel while task list is displayed', async () => {
+			expect( await homeScreen.isTaskListDisplayed() ).toBe( true );
+			expect( await homeScreen.isActivityPanelShown() ).toBe( false );
+		} );
+
+		it( 'should not show panels when there are no orders or products yet with task list hidden', async () => {
+			await homeScreen.hideTaskList();
+			expect( await homeScreen.isTaskListDisplayed() ).toBe( false );
+			expect( await homeScreen.isActivityPanelShown() ).toBe( false );
+		} );
+
+		it( 'should show Reviews panel when we have at-least one product', async () => {
+			await createSimpleProduct( simpleProductName, '9.99' );
+			await page.reload( {
+				waitUntil: [ 'networkidle0', 'domcontentloaded' ],
+			} );
+			const activityPanels = await homeScreen.getActivityPanels();
+			expect( activityPanels.length ).toBe( 1 );
+			expect(
+				activityPanels.findIndex( ( p ) => p.title === 'Reviews' )
+			).toBeGreaterThanOrEqual( 0 );
```

Comment:
```suggestion
			expect( activityPanels ).toContain(
				expect.objectContaining( { title: 'Reviews' } )
			);
```
To get something more descriptive than "Expected -1 to be greater than on equal 0"

Reply:
Nice another one that I was not are off 👍